docs/platform/tokyo2-migration/index.md

```diff
@@ -0,0 +1,128 @@
+
+
+### Clone your Linodes
+
+Because migrating a Linode will power it down during the migration, your Linode's services will be offline during that time. Also, if you run a cluster of multiple Linodes for a service, then migrating one or more of them may cause problems for the cluster.
+
+To avoid these issues, you can instead create new Linodes in Tokyo 2 and then **clone** your Tokyo 1 Linode's disks to them. Performing a clone will create exact copies of your disks on the new Linode.
+
+{{< note >}}
+You can perform a clone of a Linode when it is running. However, this can sometimes result in filesystem inconsistencies on the target Linode (your source Linode will never be negatively affected, even if the clone fails). You may need to power your source Linode down to perform a successful clone.
+{{< /note >}}
+
+Cloning your Linodes offers these benefits:
+
+-   You will be able to set up your new Linodes in Tokyo 2 and verify that they run normally before you remove your Tokyo 1 Linodes.
+
+-   When you have verified that your Tokyo 2 Linodes work, you can update your DNS records. Updating your DNS records will gracefully direct your users to your new servers without downtime.
+
+To clone a Linode, follow these steps:
+
+1.  Create a new Linode in Tokyo 2. Make sure that you [select a plan](/pricing) that has as much disk space as the size of your disks on your source Tokyo 1 Linode.
+
+1.  Follow the [cloning guide](/docs/platform/disk-images/clone-your-linode/) to complete the clone operation. When following these steps, enable all of the configuration profile options for your Linode.
+
+After you have created your new Linode and completed the clone, you may need to update your new Linode's [network configuration]((#update-your-network-configuration)). After making sure that your new Tokyo 2 servers all work as expected, [update your DNS records](#update-dns-records).
+
+{{< note >}}
+If you clone your Tokyo 1 Linodes to Tokyo 2, your Tokyo 1 Linodes will remain running and active on your account by default. To prevent double-billing, [remove](/docs/platform/billing-and-support/billing-and-payments/#removing-services) the original Tokyo 1 Lindoes after you have finished your clones. The three-month credit that will be applied to your account will cover costs of your extra servers during this transition period. This credit will not cover future costs if you do not delete those original Linodes.
+{{< /note >}}
+
+### Migrate when Scheduled
+
+If you do not choose to migrate or clone early, Linode will automatically start your migrations at the time listed in your support ticket.
+
+{{< caution >}}
+If Linode initiates your migration when it is scheduled, **your Linode will not be powered on automatically when the migration finishes**. Your Linode is not powered on in order to minimize potential security issues that could result from booting under a new IP assignment.
```

Comment:
This decision was made by the Tokyo 2 planning group, from my notes I believe @tkulick specifically called for this implementation